chore: client certificates api review (#31826)
microsoft/playwright: docs/src/api/params.md

## context-option-clientCertificates
- `clientCertificates` <[Array]<[Object]>>

>   - `origin` <[string]> Glob pattern to match against the request origin that the certificate is valid for.

Reviewer: ```suggestion
  - `origin` <[string]> Glob pattern that matches origins requiring this certificate, for example `"https://*.example.com"`.
```

How do I match `example.com` and `*.example.com` with a glob? Is glob the best pattern we can use here, given that we don't match the path?

Author: TBD.
